src/app/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import i18next from 'i18next';
+import { useForm } from 'react-hook-form';
+
+import styles from './styles.module.scss';
+import WoloxImg from './assets/wolox-logo.png';
+
+interface User {
+  first_name: string;
+  last_name: string;
+  email: string;
+  password: string;
+  password_confirmation?: string;
+  locale?: string;
+}
+
+const SignUp: React.FC = () => {
+  const { register, handleSubmit, watch, errors } = useForm<User>({ mode: 'all' });
+  const onSubmit = handleSubmit(data => {
+    data.locale = i18next.language;
+    console.log(data);
+  });
+
+  return (
+    <div className="row center">
+      <form onSubmit={onSubmit} className={`column ${styles.signupForm}`}>
+        <div className={styles.header} />
+        <div className={styles['wolox-logo-wrapper']}>
```

Comment:
css modules allows you to have the class `.wolox-logo-wrapper` mapped as `wolox-logo-wrapper` and `woloxLogoWrapper`. use `styles.woloxLogoWrapper`.
ALSO as a suggestion when you have a wrapper or a container, its better to name it as `logo-container`, the rule is `component|identifier-container`